pallets/subtensor/src/errors.rs

```diff
@@ -50,67 +62,65 @@ mod errors {
         InvalidSeal,
         /// the dispatch attempts to set weights on chain with where any normalized weight is more than MaxWeightLimit.
         MaxWeightExceeded,
-        /// tempo is not valid.
-        InvalidTempo,
         /// the hotkey attempts to become delegate when they are already.
```

Comment:
The hotkey is attempting to become a delegate when the hotkey is already a delegate.